modules/packages/TOML.chpl

```diff
@@ -76,6 +74,8 @@ module TomlParser {
 
   /* Prints a line by line output of parsing process */
   config const debugTomlParser = false;
+  /* Number of spaces in an indentation for JSON output */
+  config const tabSpace = 4;
```

Comment:
Maybe just make this a 'const' for now? 'tabSpace' seems like a config that could conflict with a lot of other packages.

Reply:
Sure thing